### DISCUSSION TITLE
New and improved 🏋️  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#88727] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret, striped shirt, and red pants holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

Double down on the opportunity a new year brings with some new ways to enhance your GitHub skills. Whether you’re a maintainer looking to better position your project’s Discussions categories or maybe you just want to add some razzle dazzle to your profile - this check-in has got you covered 🪩.  

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @NicolasGP01: Was seeing empty repositories, however after some self-sleuthing, they can back to share the culprit. [Click here to see what the solution was](https://github.com/orgs/community/discussions/87409#discussioncomment-8136072). 
* @the-1Riddle: Came in with [the steps and code](https://github.com/orgs/community/discussions/84755#discussioncomment-8033331) needed for @hezethegamer’s question about having their repository language show Markdown. It was an excellent answer with clear actionable steps and fellow Hubber @mishmanner agrees 👯. 
* @darkotodoric: Shared the code needed for a quick workaround on [inactive repository pins](https://github.com/orgs/community/discussions/84628) - they helped out several members and we appreciate it! 

### ⬆️ Level up: Resources to take your skills to new heights for the new year
* **Creating an engaging GitHub profile**: [A Step-by-Step Guide](https://dev.to/zanepearton/creating-an-engaging-github-profile-a-step-by-step-guide-4hfl) from @ZanePearton may help you freshen up your profile for the new year. 
* **GitHub Certifications are generally available**:  Unlock your full potential with GitHub Certifications! Earning a GitHub certification will give you the competitive advantage of showing up as a GitHub expert. [Learn more](https://github.blog/2024-01-08-github-certifications-are-generally-available/) from @ladykerr. 

### 📚 Rousing Reads and Beyond: GitHub-produced goodies 🍭
* **4 ways your discussions categories can be better optimized** 🧼: I share the measures we took to transform our [Projects and Issues category](https://github.com/orgs/community/discussions/categories/projects-and-issues) from a mess of miscategorized posts to the optimized category it is today. [Give it a read](https://github.com/orgs/community/discussions/88425). 
* **How we organize and get things done with SERVICEOWNERS**: Take CODEOWNERS and GitHub teams to the next level. Learn about how GitHub engineering solves the age old problem of who owns what. [Read on](https://github.blog/2023-12-19-how-we-organize-and-get-things-done-with-serviceowners/). 
* **Empowering Uruguay’s future workforce with AI**: During the second cycle of Git Commit Uruguay, students learned the basics of AI and built their own AI-powered projects. [Check out the blog post](https://github.blog/2024-01-11-empowering-uruguays-future-workforce-with-ai/). 

Any open source news you’ve read recently that you’d like to discuss? Share it below!

**Comment by queenofcorgis** [comment]
Welcome a new month with new opportunities to learn in our [New to GitHub check-in](https://github.com/orgs/community/discussions/103030).

**Reply by the1Riddle** [reply]
thanks for this opportunity :smile:

**Reply by queenofcorgis** [reply]
@the1Riddle thanks for participating in the community and helping our members!

**Reply by the1Riddle** [reply]
welcome
